tools/link.py

```diff
@@ -1770,9 +1770,9 @@ def get_full_import_name(name):
     # What you need to do is different depending on the kind of EH you use
     # (https://github.com/emscripten-core/emscripten/issues/17115).
     settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE += ['$getExceptionMessage', '$incrementExceptionRefcount', '$decrementExceptionRefcount']
-    settings.EXPORTED_FUNCTIONS += ['getExceptionMessage', '___get_exception_message', '_free']
+    settings.EXPORTED_FUNCTIONS += ['getExceptionMessage', '$incrementExceptionRefcount', '$decrementExceptionRefcount']
```

Comment:
Why do we need to add '$incrementExceptionRefcount' and '$decrementExceptionRefcount' here?

Reply:
I thought maybe they were useful for the end user the call?    Indeed the docs say:

```
Setting this option also adds refcount increasing and decreasing functions          
('incrementExceptionRefcount' and 'decrementExceptionRefcount') in the JS           
library because if you catch an exception from JS, you may need to manipulate       
the refcount manually not to leak memory. 
```

I seems that exporting them here is more consistent with the docs.   I don't see any reason to include them but then *not* export them anyway, do you?

Reply:
Yeah it looks it makes more sense to export them.